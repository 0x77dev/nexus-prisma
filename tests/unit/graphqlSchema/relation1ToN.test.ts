import { Specs } from '../../specs'
import { testGraphqlSchema } from '../../__helpers__'

testGraphqlSchema(Specs.relation1ToNReverse)
testGraphqlSchema(Specs.relation1ToNReverseAndOptional)
