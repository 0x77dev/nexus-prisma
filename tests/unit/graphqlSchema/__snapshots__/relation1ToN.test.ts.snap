// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can project user-to-posts relationship in reverse (access use via post author field). If Post.author IS optional than it IS nullable in the GraphQL API.: graphqlSchema 1`] = `
"
type Query {
  posts: [Post!]!
}

type User {
  id: ID!
}

type Post {
  id: ID!
  author: User
}
"
`;

exports[`can project user-to-posts relationship in reverse (access use via post author field). If Post.author is NOT optional than it is NOT nullable in the GraphQL API.: graphqlSchema 1`] = `
"
type Query {
  posts: [Post!]!
}

type User {
  id: ID!
}

type Post {
  id: ID!
  author: User!
}
"
`;
